src/session.rs

```diff
@@ -332,11 +336,8 @@ impl Client {
     }
 
     /// Issue the specified [`WebDriverCompatibleCommand`] to the WebDriver instance.
-    pub async fn issue_cmd(
-        &self,
-        cmd: impl WebDriverCompatibleCommand + Send + 'static,
-    ) -> Result<Json, error::CmdError> {
-        self.issue(Cmd::WebDriver(Box::new(cmd))).await
+    pub async fn issue_cmd(&self, cmd: Wcmd) -> Result<Json, error::CmdError> {
```

Comment:
I think this specifically needs to _not_ change, otherwise we're putting `Wcmd` back into the public API here.

Reply:
Apologies for the oversight. I'll revise it to use the `WebDriverCompatibleCommand` trait.